src/server/wtr-config.js

```diff
@@ -247,18 +247,19 @@ export class WTRConfig {
 	}
 
 	getBrowsers(browsers) {
-		browsers = (this.#requestedBrowsers || browsers || DEFAULT_BROWSERS).map(b => BROWSER_MAP[b] || 'chromium');
+		browsers = (this.#requestedBrowsers || browsers || DEFAULT_BROWSERS).map(b => BROWSER_MAP[b] || BROWSER_MAP.chrome);
 
 		if (!Array.isArray(browsers)) throw new TypeError('browsers must be an array');
 
 		return [...new Set(browsers)].map((b) => playwrightLauncher({
-			concurrency: b === 'firefox' || this.#cliArgs.open ? 1 : undefined, // focus in Firefox unreliable if concurrency > 1 (https://github.com/modernweb-dev/web/issues/238)
+			concurrency: b === BROWSER_MAP.firefox || this.#cliArgs.open ? 1 : undefined, // focus in Firefox unreliable if concurrency > 1 (https://github.com/modernweb-dev/web/issues/238)
 			product: b,
 			createBrowserContext: ({ browser }) => browser.newContext({ deviceScaleFactor: 2, reducedMotion: 'reduce' }),
 			launchOptions: {
 				headless: !this.#cliArgs.open,
 				devtools: false,
-				slowMo: this.#cliArgs.slowmo || 0
+				slowMo: this.#cliArgs.slowmo || 0,
+				args: b === BROWSER_MAP.chrome ? ['--disable-gpu-rasterization'] : []
```

Comment:
`--disable-gpu-rasterization` seems to work just as well as `--disable-gpu` and `--disable-partial-raster` together, and seems better than disabling all hardware acceleration.